sources/packages/api/src/route-controllers/institution-locations/institution-location.institutions.controller.ts

```diff
@@ -217,4 +217,33 @@ export class InstitutionLocationInstitutionsController extends BaseController {
     const offering = application.currentAssessment.offering;
     return transformToActiveApplicationDataAPIOutDTO(application, offering);
   }
+
+  /**
+   * Get locations details of logged in user.
+   * @returns location details.
+   */
+  @Get()
+  async getMyInstitutionLocationsDetails(
+    @UserToken() userToken: IInstitutionUserToken,
+  ): Promise<InstitutionLocationsAPIOutDTO[]> {
+    // Get all institution locations that user has access too.
+    const InstitutionLocationData =
+      await this.locationService.getMyInstitutionLocations(
+        userToken.authorizations.getLocationsIds(),
+      );
+    return InstitutionLocationData.map((location) => {
+      return {
+        id: location.id,
+        name: location.name,
+        address: {
+          addressLine1: location.data.address?.addressLine1,
+          addressLine2: location.data.address?.addressLine2,
+          provinceState: location.data.address?.provinceState,
```

Comment:
just a qn, we do have a different story to update to the new address format in other forms right?

Reply:
I am not sure. Maybe during a UI review but I do not believe that we have something specific.

Reply:
Apparently, this address is returned only to be saved to the Vue store but from there is not consumed anywhere. I would be considering removing it. Thoughts?

Reply:
oh..okay... if the endpoint is only for the state, then it's fine

Reply:
But I am still keen to remove the address part. If we need it in the future we can add it. I do not recall why we added it.
My best guess is that old UI maybe have some address information in the headers.

Reply:
@ann-aot I removed the address part from this method and from the state also - FYI 😉